README.md

```diff
@@ -1 +1,33 @@
 # gazebo_ros2_control
+
+This repository contains the contents for testing gazebo_ros2_control
+
+It is running Gazebo and some other ROS 2 nodes.
+
+## Video + Pictures
+
+![](img/gazebo_ros2_control_position_pid.gif)
+
+## Running
+
+### Modifying or building your own
+
+```bash
+cd Docker
+docker build -t gazebo_ros2_control_docker .
+```
+
+### To run the demo
+
+The following command will launch Gazebo
+
+```bash
+rocker --x11 --nvidia gazebo_ros2_control:latest
```

Comment:
Is `rocker` a fancy new docker tool I'm not aware of?? :D 
It may be worth dropping a note on how to install/configure *or* fixing the typo

Reply:
rocker is fancy osrf tool https://github.com/osrf/rocker

Reply:
I will add how to install it in the documentation